privacyidea/lib/periodictask.py

```diff
@@ -111,6 +112,8 @@ def set_periodic_task(name, interval, nodes, taskmodule, ordering=0, options=Non
     :type options: Dictionary mapping unicodes to values that can be converted to unicode or None
     :param active: Flag determining whether the periodic task is active
     :type active: bool
+    :param retry_if_failed: true if a cron job will be retryed by failing
```

Comment:
Please change this to s.t.h. like 

    true if the privacyidea should retry to execute this periodic task if it fails

Reply:
Done in 969450a5